apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -228,6 +248,36 @@ defmodule OMG.Eth.RootChain do
          do: {:ok, Enum.map(logs, &decode_exit_started/1)}
   end
 
+  defp get_inflight_data(hash) do
+    {:ok, eth_tx} = Ethereumex.HttpClient.eth_get_transaction_by_hash(hash)
+    encode_inflight = Eth.Encoding.from_hex(eth_tx["input"])
+
+    ABI.decode(
+      ABI.FunctionSelector.parse_specification_item(%{
+        "type" => "function",
+        "name" => "startInFlightExit",
+        "inputs" => List.duplicate(%{"type" => "bytes"}, 4),
+        "outputs" => []
+      }),
+      encode_inflight
+    )
+  end
+
+  @doc """
+  Returns InFlightExit from a range of blocks.
+  """
+  def get_in_flight_exit_starts(block_from, block_to, contract \\ nil) do
+    contract = contract || from_hex(Application.get_env(:omg_eth, :contract_addr))
+    signature = "InFlightExitStarted(address,bytes32)"
+
+    with {:ok, logs} <- Eth.get_ethereum_events(block_from, block_to, signature, contract) do
+      {:ok,
+       Enum.map(logs, fn log ->
+         get_inflight_data(log["transactionHash"])
```

Comment:
not sure, but it looks like the result of `get_in_flight_exit_starts` differs a lot from the other "get_logs" kinds of functions. I think its worth making it consistent, i.e. 
1/ return a map with the event fields
2/ I think also `eth_height` is getting included everywhere, we'll need it anyway, so it's worth to be consistent
3/ include the `get_inflight_data` data as a special field to this map

This will also have the benefit of a more sensible type [here](https://github.com/omisego/elixir-omg/pull/352/files#diff-a19f0c837d4bfde56a9db59968310a30R47) and a more readable pattern match [here](https://github.com/omisego/elixir-omg/pull/352/files#diff-a19f0c837d4bfde56a9db59968310a30R425)